server/app.js

```diff
@@ -50,6 +52,6 @@ app.use(function(err, req, res, next) {
 
 //listening to this port 
 const PORT = process.env.PORT || 4000;
```

Comment:
maybe make a note in the `server` readme file to notice whoever running the app that they perhaps want to put in an env var for PORT or, by default it's 4000

Reply:
Sure Bonnie.